superset/assets/spec/javascripts/explorev2/components/FieldSetRow_spec.js

```diff
@@ -25,7 +20,6 @@ describe('FieldSetRow', () => {
   });
 
   it('renders a FieldSet for each item in fieldSets array', () => {
-    const length = defaultProps.fieldSets.length;
-    expect(wrapper.find(FieldSet)).to.have.lengthOf(length);
```

Comment:
The new component is more generic than the previous one as I moved much of the logic in the parent component. Now it really just makes a row out of an array of arbitrary nodes. I'll fix up the naming and add a few more test cases.